CHANGELOG.md

```diff
@@ -27,6 +27,14 @@ Ansible Changes By Release
   the old overwriting behaviour via the config option.  In 2.5, multiple --tags
   options will be merged with no way to go back to the old behaviour.
 
+###Modules Notes:
```

Comment:
I'm not sure if most users don't stop reading at potentially huge list of new modules and will ever get to "Modules notes" section if it's after "New modules" section